package.json

```diff
@@ -2,7 +2,7 @@
   "name": "root",
   "private": true,
   "engines": {
-    "node": "10.x"
+    "node": ">=10.x"
```

Comment:
It would be great to just move to the LTS (currently 12) version. For that we would also need to change travis.yml and Dockerfile.
This would allow us to use some features not present in node 10, like `Array.prototype.{flat, flatMap}` for example (I've seen reduce-based implementation of it somewhere in the code :D

Reply:
yeah, I think that if we can we should start with lts :man_shrugging: